grpc_client.go

```diff
@@ -9,28 +9,28 @@ import (
 	"fmt"
 	"math"
 	"net"
-	"time"
 
 	"github.com/hashicorp/go-plugin/internal/plugin"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/health/grpc_health_v1"
 )
 
-func dialGRPCConn(tls *tls.Config, dialer func(string, time.Duration) (net.Conn, error), dialOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
+func dialGRPCConn(tls *tls.Config, dialer func(context.Context, string) (net.Conn, error), dialOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
```

Comment:
This and other change below is to fix a deprecation notice. Check 1ca82c20b153682791df7290ef9264fc8e5f9634's commit message for further details.